oshi-core/src/test/java/oshi/hardware/GlobalMemoryTest.java

```diff
@@ -49,4 +47,27 @@ public void testGlobalMemory() {
         assertTrue(memory.getSwapUsed() >= 0);
         assertTrue(memory.getSwapUsed() <= memory.getSwapTotal());
     }
+
+    /**
+     * Test OSProcess setters and getters
+     */
+    @Test
+    public void testGlobalMemoryCopy() {
+        SystemInfo si = new SystemInfo();
+        HardwareAbstractionLayer hal = si.getHardware();
+        GlobalMemory oldMemory = hal.getMemory();
+        assertNotNull(oldMemory);
+
+        DefaultAbstractGlobalMemory newMemory = new DefaultAbstractGlobalMemory();

```

Comment:
You've jumped through some hoops to create a new empty class so you can test the setters and getters, but why not just test arbitrary values like 42 or 12345.  You can test those in the same method you use to test for sane values for the other memory attributes.